Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM        sdurrheimer/alpine-golang-make-onbuild
+FROM        prom/busybox:glibc
 MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
 
-EXPOSE      9100
+COPY node_exporter  /bin/node_exporter
+
+EXPOSE     9100
+ENTRYPOINT [ "/bin/node_exporter" ]
```

Comment:
An entrypoint makes sense when there is mandatory setup that needs to
happen or an environment needs to be set up dynamically before *any*
command can be run. For the command to be run itself, I find entrypoints
way too heavy handed because it's basically impossible to override. Use CMD
to specify the default action, it leaves the option to do something else,
like get a shell in the same container.